### PR TITLE
Feature/implement voice chat

### PR DESCRIPTION
**Pull request by Daboo1998**

### CODE REVIEW

File: packages/app/src/scenes/home/pages/HomePage/components/templates/SocialPanel/SocialPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  name: string;
+  main: () => JSX.Element | null;
+}
+
+// TODO: When implementing Chat, move this to seperate tab and implemnt it in it
+const Chat: FC = () => {
+  return <div>Chat</div>;
+};
+
+const SocialPanel: FC = () => {
```
Comment (PAKovalev) [inline packages/app/src/scenes/home/pages/HomePage/components/templates/SocialPanel/SocialPanel.tsx]: It is widget!

File: packages/app/src/scenes/App.routes.tsx [hunk 2]
```diff
@@ -164,11 +164,6 @@ export const CORE_ROUTES: RouteConfigInterface[] = [
 ];
 
 export const PRIVATE_ROUTES_WITH_UNITY: RouteConfigInterface[] = [
-  {
```
Comment (PAKovalev) [inline packages/app/src/scenes/App.routes.tsx]: revert back.

File: packages/app/src/scenes/App.tsx [hunk 3]
```diff
@@ -205,7 +205,10 @@ const App: FC = () => {
             <GlobalStyles />
             <UnityPage />
             <Suspense fallback={false}>
-              <AppLayers>{createSwitchByConfig(PRIVATE_ROUTES_WITH_UNITY, ROUTES.base)}</AppLayers>
+              <AppLayers>
+                <HomePage />
```
Comment (PAKovalev) [inline packages/app/src/scenes/App.tsx]: Remove!

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: packages/app/src/scenes/home/pages/HomePage/HomePage.tsx [hunk 4]
```diff
@@ -29,6 +31,10 @@ const HomePage: FC = () => {
   //   [unityStore, history]
   // );
 
+  if (objectStore.asset?.isExpanded === true) {
+    return null;
+  }
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/home/pages/HomePage/HomePage.tsx]: Is this actually needed?

File: packages/app/src/scenes/widgets/pages/SocialWidget/SocialWidget.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        name: 'Voice',
+        main: () => <VoicePanel />
+      }
+    ],
+    []
+  );
+
+  const [selectedTabIndex, setSelectedTabIndex] = useState(1);
+
+  const selectedTab = useMemo(() => tabs[selectedTabIndex], [selectedTabIndex, tabs]);
```
Comment (dmitry-yudakov) [inline packages/app/src/scenes/widgets/pages/SocialWidget/SocialWidget.tsx]: no need of memo here

**Review by dmitry-yudakov — APPROVED** [review]

**Review by PAKovalev — APPROVED** [review]

File: packages/app/src/scenes/home/pages/HomePage/HomePage.tsx (hunk 4 above)
Comment (Daboo1998) [reply packages/app/src/scenes/home/pages/HomePage/HomePage.tsx]: Now not

**Pull request merged**